lib/ios-deploy.js

```diff
@@ -80,7 +80,7 @@ class IOSDeploy {
             writeStream.on('error', reject);
           });
           readStream.pipe(writeStream);
-          await itemPushWait.timeout(`Couldn't push '${itemPath}' within the timeout ${ITEM_PUSH_TIMEOUT}ms`, ITEM_PUSH_TIMEOUT);
+          await itemPushWait.timeout(ITEM_PUSH_TIMEOUT, `Couldn't push '${itemPath}' within the timeout ${ITEM_PUSH_TIMEOUT}ms`);
```

Comment:
actually, the second argument should be an error instance. Please also wrap the string into `new Error()`